.github/ISSUE_TEMPLATE/package_issue.yaml

```diff
@@ -0,0 +1,29 @@
+name: Report package issues
```

Comment:
Don't we have already https://github.com/thoth-station/support/issues/new?assignees=&labels=bug%2Ctriage&template=resolver_issue.yaml?

Reply:
That was meant to be specific for resolution failures. This is specific to issues one can observe when using different packages - so we can provide prescriptions for them. Not sure if the "prescriptions" repository will have the same attention as "support".